### ISSUE TITLE
Stories without full text transcripts should be indicated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tripodtr/npr-android-app#9]
```
Since I primarily read the news, I'd like to know when a story has a full 
transcript vs. just a summary vs. nothing at all.

To find an example of this, check out many of the Program entries.  I propose 
some sort of icon to distinguish between the different story types.
```

Original issue reported on code.google.com by `mark.mas...@gmail.com` on 4 Mar 2010 at 4:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `jpenn...@gmail.com` on 21 Jun 2010 at 8:36
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Just to clarify, from an API perspective does "full text" mean that it has 
story text with paragraphs, and ones that don't just have a miniTeaser or 
teaser?

This should be easy enough to implement technically. The user experience might 
be a little more challenging. How does an icon signify "only a teaser and 
audio"? And how does that icon work with / layout with the audio icon there now?
```

Original comment by `jeremy.w...@gmail.com` on 23 Mar 2011 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
No clarification, so putting in backlog.
```

Original comment by `jpenn...@gmail.com` on 19 May 2011 at 7:33
- Changed state: **Backlog**

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed as of the 2.1 release.
```

Original comment by `jpenn...@gmail.com` on 18 Oct 2011 at 7:48
- Changed state: **Fixed**
